src/qt_gui/keyboardcontrolswindow.cpp

```diff
@@ -0,0 +1,620 @@
+    QObject::connect(ui->DPadDownkeySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+    QObject::connect(ui->L2keySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+    QObject::connect(ui->L1keySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+    QObject::connect(ui->CrossKeySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+    QObject::connect(ui->R2KeySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+    QObject::connect(ui->CircleKeySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+    QObject::connect(ui->R1KeySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+    QObject::connect(ui->SquareKeySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+    QObject::connect(ui->TriangleKeySequenceEdit, &QKeySequenceEdit::editingFinished, this,
+                     &KeyboardControlsWindow::onEditingFinished);
+
+    m_listOfKeySequenceEdits.insert(ui->StartKeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->SelectKeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->LAnalogDownkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->LAnalogLeftkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->LAnalogUpkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->LAnalogRightkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->PSkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->RAnalogDownkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->RAnalogLeftkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->RAnalogUpkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->RAnalogRightkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->DPadLeftkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->DPadRightkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->DPadUpkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->DPadDownkeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->L2keySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->L1keySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->CrossKeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->R2KeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->CircleKeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->R1KeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->SquareKeySequenceEdit);
+    m_listOfKeySequenceEdits.insert(ui->TriangleKeySequenceEdit);
+
+    m_listOfKeySequenceEdits.insert(ui->StartKeySequenceEdit);
```

Comment:
This looks duplicated

Reply:
oopps.. probably as the result of fixing conflicts during a rebase (nice catch!)